aten/src/ATen/native/Loss.cpp

```diff
@@ -123,6 +123,21 @@ Tensor kl_div_backward_cpu(const Tensor& grad, const Tensor& input, const Tensor
   return grad_input;
 }
 
+Tensor cross_entropy_loss_with_soft_labels(
+    const Tensor& self,
+    const Tensor& target,
+    int64_t reduction) {
+  TORCH_CHECK(self.dim() == 2, "Expected 2D input; got shape: ", self.sizes());
```

Comment:
nit: Say the name of the function: "cross_entropy_loss_with_soft_labels(input, target): Expected 2d input..."

Reply:
From the [TF docs](https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy), it supports shape: `[batch_size, d0, .. dN]`. I'll remove the 2-dim constraint here as well since there's no reason for it (both components support > 2 dims).